src/ontology/mondo-edit.obo

```diff
@@ -412091,6 +412151,22 @@ synonym: "immunodeficiency 75" EXACT [OMIM:619126]
 xref: OMIM:619126 {source="MONDO:equivalentTo"}
 is_a: MONDO:0021094 {source="OMIM:619126"} ! immunodeficiency disease
 
+[Term]
+id: MONDO:0030859
+name: COACH syndrome 2
+synonym: "COACH2" EXACT ABBREVIATION [OMIM:619111]
+xref: OMIM:619111 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```

Comment:
where is generic COACH?

Reply:
It looks like OMIM has not created a PS yet to group the 3 subtypes of COACH syndrome. 

COACH SYNDROME 1; COACH1 https://omim.org/entry/216360
COACH SYNDROME 2; COACH2 https://omim.org/entry/619111
COACH SYNDROME 3; COACH3 https://omim.org/entry/619113

Reply:
I will create a generic grouping class